embedded-graphics/src/fonts/text.rs

```diff
@@ -293,4 +304,36 @@ mod tests {
 
         Ok(())
     }
+
+    #[test]
+    fn multiline() -> Result<(), core::convert::Infallible> {
+        let mut display = MockDisplay::new();
+
+        Text::new("##\n#", Point::zero())
+            .into_styled(TextStyle::new(SpacedFont, BinaryColor::On))
+            .draw(&mut display)?;
+
+        assert_eq!(
+            display,
+            MockDisplay::from_pattern(&[
+                "####     ####",
+                "# #      # # ",
+                " # #      # #",
+                "   #        #",
+                "####         ",
+                "# #          ",
+                " # #         ",
+                "   #         ",
+            ])
+        );
+
+        assert_eq!(
+            Text::new("#\n#", Point::zero())
+                .into_styled(TextStyle::new(SpacedFont, BinaryColor::On))
+                .size(),
+            Size::new(4, 8)
+        );
+
+        Ok(())
+    }
```

Comment:
> Can you modify this test to use a non-spaced font, maybe `Font6x8`? If you think spacing _and_ multiline should be tested simultaneously, please add a new test that does that.
> 
> I'd also like to see a better character used than `#` - it's difficult to make out in the `MockDisplay`. How about using `AB\nC` as the test string?

This is indeed more readable with `AB\nC` and `Font6x8`, so I modified to the patch to use this.

Reply:
> I also noticed there's no spacing between lines. Should we add a `LINE_SPACING` property to `Font` to cater for this? It should probably default to `1` (pixel).

Most fonts have one blank line under the characters (except a few characters) so multiline text is readable anyway.

Still, I also thought about being able to modify the vertical spacing between lines of text, but I think it would be useful to be able to specify it for each Text instance rather than having it as a property of a Font. I haven't looked at it yet but I guess that it could be specified inside a TextStyle.

Reply:
> Most fonts have one blank line under the characters (except a few characters) so multiline text is readable anyway.

That's true. Let's leave it as is for now then.

> I haven't looked at it yet but I guess that it could be specified inside a TextStyle.

Yes, I actually opened #268 to come to a more general solution. We can keep this PR small for now and add to it later.